eng/mgmt/automation/generate_utils.py

```diff
@@ -77,7 +78,9 @@ def generate(
         )
     logging.info(command)
     if os.system(command) != 0:
-        logging.error('[GENERATE] Autorest fail')
+        error_message = '[GENERATE][Error] Autorest fail'
```

Comment:
Can we add `Next Steps` information to all the error message in general to let the user know what are the expected actions to take for them?

Reply:
Currently it is hard to say what to do next.

My personal rule of thumb (before checking the CI output) is:
If all/most SDK CI fails, that usually because Swagger definition is wrong (e.g. error from autorest or m4).
If only Java fails, it may be from Java codegen.

Reply:
Hopefully TypeSpec could be better on this regard.

If "TypeSpec Validation" is green, we can assume TypeSpec source is good.